code/modules/mob/living/simple_animal/abnormality/waw/snow_whites_apple.dm

```diff
@@ -213,12 +222,27 @@ GLOBAL_LIST_EMPTY(vine_list)
 			/mob/dead,
 			/mob/living/simple_animal/hostile/abnormality/snow_whites_apple))
 
-	last_expand = world.time + growth_cooldown_low
-
-/obj/structure/alien/weeds/apple_vine/set_base_icon()
-	return
+/obj/structure/apple_vine/Crossed(atom/movable/AM)
+	. = ..()
+	if(is_type_in_typecache(AM, atom_remove_condition))
+		take_damage(15, BRUTE, "melee", 1)
+	if(is_type_in_typecache(AM, ignore_typecache))		// Don't want the traps triggered by sparks, ghosts or projectiles.
+		return
+	if(isliving(AM))
+		vine_effect(AM)
 
-/obj/structure/alien/weeds/apple_vine/expand()
+/obj/structure/apple_vine/play_attack_sound(damage_amount, damage_type = BRUTE, damage_flag = 0)
+	switch(damage_type)
+		if(BRUTE)
+			if(damage_amount)
+				playsound(loc, 'sound/effects/attackblob.ogg', 100, TRUE)
+			else
+				playsound(src, 'sound/weapons/tap.ogg', 50, TRUE)
+		if(BURN)
```

Comment:
I can see it is copied from blob.
We do not use brute or burn damage types, so this won't work.

Reply:
This was due to the fact when structures are hit they make a metal sound. This is actually taken from Alien Weeds.

Reply:
I will change it to be only one sound when hit though.